File: Modul-6/soal-1/PRAK601-2410817210017-OrlandoSugian.py
brs, klm = map(int, input("").split())
inputan = list(map(int, input("").split()))
matriks = [inputan[i * klm:(i + 1) * klm] for i in range(brs)]
for row in matriks:
    print(" ".join(map(str, row)))

File: Modul-6/soal-1/PRAK601-2410817210017-OrlandoSugian.c
#include <stdio.h>
int main() {
    int brs, klm;
    scanf("%d %d", &brs, &klm);
    int matriks[brs][klm];
    for (int i = 0; i < brs; i++) {
        for (int j = 0; j < klm; j++) {
        scanf("%d", &matriks[i][j]);
        }
    }
    printf("");
    for (int i = 0; i < brs; i++) {
        for (int j = 0; j < klm; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}
